bonsai/sdk/Cargo.toml

```diff
@@ -15,7 +15,7 @@ tokio = { version = "1", features = ["full", "sync"], optional = true }
 
 [dev-dependencies]
 env_logger = "0.10"
-httpmock = "0.6"
+httpmock = "0.6.8"
```

Comment:
Why is this change needed? Typically I ignore patch versions

Reply:
Removed the patch version in 4d19fe86412bca8d17f87f872c0808d0074511a3